ext/settings.html

```diff
@@ -1581,6 +1581,47 @@ <h1>Yomichan Settings</h1>
                 <button class="icon-button"><span class="icon-button-inner"><span class="icon" data-icon="material-right-arrow"></span></span></button>
             </div>
         </div></div>
+        <div class="settings-item advanced-only">
+            <div class="settings-item-inner">
+                <div class="settings-item-left">
+                    <div class="settings-item-label">Show custom card tags</div>
+                    <div class="settings-item-description">
+                        Show an icon next to the "view added note" button that shows all tags on that note (if it has
+                        any).
+                    </div>
+                </div>
+                <div class="settings-item-right">
+                    <label class="toggle">
```

Comment:
Use the single-line layout for toggles (used elsewhere), otherwise it adds a lot of extra noise. (It would be nice if [web components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) could be used so it's just a single element, but the minimum browser versions prevent this sadly, hence the repetition)

(My comment above in this file may render this feedback unnecessary)